src/web/src/asset_POST.ecpp

```diff
@@ -63,7 +64,7 @@ bool database_ready;
     }
     catch (const std::exception& e) {
         LOG_END_ABNORMAL(e);
-        http_die("bad-request-document", e.what());
+        http_die ("bad-request-document", TRANSLATE_ME (e.what()));
```

Comment:
I'm not sure this might work, as e.what already contains some message we can't get here, As discussed with Jana on IRC, we should use JSON_ME or JSONIFY_ME, or whatever she creates for the purpose (UI might translate the string later, but it will be a problem to get the actual string)